deploy/webhook/cloudbuild.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
+  # Build and push the image with the :commit_sha tag
+- name: gcr.io/k8s-skaffold/skaffold:v0.16.0
+  args:
+  - 'skaffold'
+  - '-f'
+  - 'deploy/webhook/skaffold.yaml'
+  - 'run'
+  - '--tag'
+  - '${COMMIT_SHA}'
```

Comment:
one last nit: why don't you let skaffold compute the commit sha! thats what the gitTagger does and its the default

Reply:
this PR is becoming an advertisement for the power of skaffold :)

Reply:
LOL solid i had no idea! hypeee